.circleci/config.yml

```diff
@@ -1,11 +1,42 @@
-version: 2
+version: 2.1
+
+commands:
+    configure-composer-auth:
+        description: Configure composer auth
+        steps:
+            - run: mkdir -p ~/.composer/cache
+            - run:
+                name: Add GitHub token for composer
+                command: echo ${COMPOSER_AUTH} > ~/.composer/auth.json
+
+    restore-composer-cache:
+        description: Restore Composer cache
+        steps:
+            - restore_cache:
+                name: Restore Composer cache
+                keys:
+                    - composer-{{ .Branch }}
+                    - composer-
+
+    save-composer-cache:
+        description: Save Composer cache
+        steps:
+            - save_cache:
+                name: Save Composer cache
+                key: composer-{{ .Branch }}
+                paths:
+                    - ~/.composer/cache
+
 jobs:
     build:
         machine:
-            image: ubuntu-1604:201903-01
+            image: ubuntu-1604:202004-01
```

Comment:
:+1: I didn't know about that btw

Reply:
Yep, they released it 2 weeks ago or something.